setup.py

```diff
@@ -21,6 +21,7 @@
         'beautifulsoup4>=4.4',
         'lxml>=3.6',
         'pandas>=0.18',
+        'pytest>3.1',
```

Comment:
Hum… `pytest` is not a install requirement (`install_requires`), it's a test requirement (`tests_requires`) according to [the docs](https://setuptools.readthedocs.io/en/latest/setuptools.html?highlight=tests_require#new-and-changed-setup-keywords). This means two things:

1. `pytest` should not be installed for regular users, just for developers
2. We should run tests with `python setup.py test` if we're injecting tests dependencies

Does that make sense?